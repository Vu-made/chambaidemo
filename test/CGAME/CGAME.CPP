#include <bits/stdc++.h>
#define N 100005
using namespace std;
int a[N];
int s1 = 0, s2 = 0, n;
int main()
{
    freopen("CGAME.INP","r",stdin);
    freopen("CGAME.OUT","w",stdout);
    cin >> n;
    for (int i=1; i<=n; i++) cin >> a[i];
    int i = 1, j = n, d = 0;
    while (i <= j)
    {
        if (d == 0) s1 += max(a[i],a[j]);
        else s2 += max(a[i],a[j]);

        if (a[i] > a[j]) i++;
        else j--;
        d = 1 - d;
    }
    cout << s1 << " " << s2;
    return 0;
}
