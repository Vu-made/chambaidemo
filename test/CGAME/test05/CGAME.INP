66
18 23 34 3 37 8 54 25 49 66 27 38 40 28 61 26 57 56 5 64 12 59 11 47 22 20 65 31 53 15 17 55 44 24 32 45 58 10 46 50 43 30 1 51 48 13 2 9 35 16 29 7 62 42 39 33 19 60 21 6 14 4 63 36 41 52 