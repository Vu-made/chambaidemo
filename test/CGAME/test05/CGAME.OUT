1123 1088